modules/ROOT/pages/managing-instances/backup-restore-export.adoc

```diff
@@ -41,12 +41,18 @@ This type of snapshot is the only snapshot available for Free instances.
 image::snapshot-actions.png[width=400]
 
 [[export-create]]
-=== Export/Create
 
-From the more (*...*) menu next to an existing snapshot, you can:
+=== Export
 
-* *Export* - Download the instance as a *_.backup_* file (for latest version) or *_.dump_* (for version 4.x), allowing you to store a local copy and work on your data offline.
-* *Create instance from snapshot* - Create a new AuraDB instance using the data from the snapshot.
+From the more menu (...) next to an existing snapshot, you can *Export*.
+*Export* downloads the instance as a compressed `.backup` file allowing you to store a local copy and work on your data offline.
+For AuraDB instances (including AuraDB instances with the Graph Analytics plugin installed) backups are always exported as a `backup` file.
+
+=== Create instance from snapshot
+
+You can create an instance using the data from a snapshot, just like creating a clone of an instance.
+From the more (...) menu next to an existing snapshot select *Create instance from snapshot*.
```

Comment:
```suggestion
From the more menu (...) next to an existing snapshot, you can:
* *Export* - Download the *AuraDB* instance as a *_.backup_* file (for latest version) or *_.dump_* (for version 4.x) or the *AuraDS* instance as a *_.tar_* file, allowing you to store a local copy and work on your data offline.
* *Create instance from snapshot* - Create a new instance using the data from the snapshot.

```

Reply:
There is no need to be repetitive. Since both options are available from the same action (the ... menu), it's better to keep them together.
Also, are you sure about 4.x snapshots? I saw nothing about them on the card or in the attached Slack conversation. I merged [this](https://github.com/neo4j/docs-aura/pull/592) PR two weeks ago that contains that information. It was reviewed and approved by Alena, so better double-check on this.